query-node/mappings/src/content/nft.ts

```diff
@@ -824,7 +845,9 @@ export async function contentNft_BidMadeCompletingAuction({
   // specific event processing
 
   // create record for winning bid
-  await createBid(event, store, memberId.toNumber(), videoId.toNumber())
+  const {
+    video: { nft },
+  } = await createBid(event, store, memberId.toNumber(), videoId.toNumber())
```

Comment:
After the change of `getCurrentAuctionFromVideo` as described above, we can make `createBid` return guaranteed `nft` object as well. It will remove the need to use optional chaining used few lines below.
```
ownerMember: nft.ownerMember,
ownerCuratorGroup: nft.ownerCuratorGroup,
```